app/controllers/editions_controller.rb

```diff
@@ -331,6 +347,10 @@ def description(r)
 
 private
 
+  def redirect_url
+    params.fetch("redirect_url", "").sub(%r{^https?://(www\.)?gov\.uk/}, "/")
```

Comment:
Apols if it's obvious, but what does this do? Still haven't memorised my regex patterns :/ Would there be a benefit of putting this in an intention revealing named method?